solutions/usaco-671.mdx

```diff
@@ -84,4 +84,61 @@ int main()
 
 </CPPSection>
 
+<PySection>
+
+```py
+from collections import defaultdict, deque
+
+HORIZ, VERTI = True, False  # direction of laser beam (horizontal or vertical)
```

Comment:
i'd ditch the constants here and use them directly
add a comment like "true = horizontal, false = vertical"